src/CloudEvents/Serializers/Formatters/Formatter.php

```diff
@@ -27,9 +27,13 @@ public function encodeTime(?DateTimeInterface $time): ?string
 
     public function decodeTime(?string $time): ?DateTimeInterface
     {
-        return $time === null
-            ? null
-            : DateTimeImmutable::createFromFormat(self::TIME_FORMAT, $time, new DateTimeZone(self::TIME_ZONE));
+        $parsed = DateTimeImmutable::createFromFormat(self::TIME_FORMAT, $time, new DateTimeZone(self::TIME_ZONE));
+
+        if($parsed) {
+            return $parsed;
+        }
+        
+        return null;
```

Comment:
Going to rework this to reflect the ternary shorthand for styling.

```suggestion
        return $parsed === false
            ? null
            : $parsed;
```